BaseUI/src/screens/AddPeopleScreen.tsx

```diff
@@ -0,0 +1,150 @@
+import CheckBox from '@react-native-community/checkbox';
+import React, {useEffect, useState} from 'react';
+import {
+  Dimensions,
+  ListRenderItemInfo,
+  SectionList,
+  Text,
+  View,
+} from 'react-native';
+import {SearchBar} from 'react-native-elements';
+import {Swipeable} from 'react-native-gesture-handler';
+import {Stack} from 'react-native-spacing-system';
+import {commonStyles} from '../../styles/commonStyles';
+import {profileForm} from '../../styles/profileForm';
+import {sectionList} from '../../styles/sectionList';
+import ProfileForm from '../components/ProfileForm';
+import ProfileFormContent from '../components/ProfileFormContent';
+import ProfileFormHeader from '../components/ProfileFormHeader';
+import {usersList} from '../components/SectionListUsers';
+import SubscriberCell from '../components/SubscriberCell';
+import {SubscriberItem} from '../components/SubscriberItem';
+
+const windowHeight = Dimensions.get('window').height;
+
+const AddPeopleScreen = () => {
+  const [listData, setUsersList] = useState(usersList);
```

Comment:
```suggestion
  const [listData, setListData] = useState(usersList);
```

придерживайся ж уже однородности